x-pack/plugin/ccr/qa/multi-cluster/src/test/java/org/elasticsearch/xpack/ccr/AutoFollowIT.java

```diff
@@ -663,6 +671,80 @@ public void testDataStreamsBiDirectionalReplication() throws Exception {
         }
     }
 
+    public void testAutoFollowSearchableSnapshotsFails() throws Exception {
+        if ("follow".equals(targetCluster) == false) {
+            return;
+        }
+
+        final String testPrefix = getTestName().toLowerCase(Locale.ROOT);
+        int initialNumberOfSuccessfulFollowedIndicesInFollowCluster = getNumberOfSuccessfulFollowedIndices();
+
+        final String autoFollowPattern = "pattern-" + testPrefix;
+        createAutoFollowPattern(client(), autoFollowPattern, testPrefix + "-*", "leader_cluster");
+
+        // Create a regular index on leader
+        final String regularIndex = testPrefix + "-regular";
+        {
+            try (var leaderClient = buildLeaderClient()) {
+                for (int i = 0; i < 10; i++) {
+                    var indexRequest = new Request("POST", "/" + regularIndex + "/_doc");
+                    indexRequest.addParameter("refresh", "true");
+                    indexRequest.setJsonEntity("{\"value\":" + i + "}");
+                    assertOK(leaderClient.performRequest(indexRequest));
+                }
+                verifyDocuments(leaderClient, regularIndex, 10);
+            }
+        }
+
+        // Create a snapshot backed index on leader
+        final String mountedIndex = testPrefix + "-mounted";
+        {
+            try (var leaderClient = buildLeaderClient()) {
+                final String repositoryPath = System.getProperty("tests.leader_cluster_repository_path") + '/' + testPrefix;
+                assertThat("Missing system property [tests.leader_cluster_repository_path]", repositoryPath, not(emptyOrNullString()));
```

Comment:
I think you meant to check the value of the system property and not the value of the concatenated string (which cannot be empty).

Reply:
I pushed a2a5fdfb14426f9d923bb75eb242162176b02eaa